src/components/SummaryPanel/RpsChart.tsx

```diff
@@ -94,33 +126,72 @@ export default class RpsChart extends React.Component<RpsChartTypeProp, {}> {
   };
 
   private renderSparkline = () => {
-    if (!this.thereIsTrafficData()) {
+    if (!thereIsTrafficData(this.props.dataRps)) {
       return null;
     }
 
-    const chartData = {
-      x: 'x',
-      columns: (this.props.dataRps as [string, number][]).concat(this.props.dataErrors as [string, number][]),
-      type: 'area-spline'
-    };
+    return renderSparkline(this.props.dataRps.concat(this.props.dataErrors), [PfColors.Blue, PfColors.Red]);
+  };
+}
 
+export class TcpChart extends React.Component<TcpChartTypeProp, {}> {
+  render() {
     return (
-      <AreaChart
-        size={{ height: 45 }}
-        color={{ pattern: [PfColors.Blue, PfColors.Red100] }}
-        legend={{ show: false }}
-        grid={{ y: { show: false } }}
-        axis={sparklineAxisProps}
-        data={chartData}
-      />
+      <div className={blockStyle}>
+        <div>
+          <strong>{this.props.label} - min / max:</strong>
+        </div>
+        {this.thereIsTrafficData() ? this.renderMinMaxStats() : renderNoTrafficLegend()}
+        {this.renderSparkline()}
+      </div>
     );
-  };
+  }
+
+  private renderMinMaxStats = () => {
+    let dataSent: any = [],
+      dataReceived: any = [];
+    if (this.props.sentRates.length > 0) {
+      // NOTE: props.sentRates and props.receivedRates are arrays of data value points EXCEPT for the first array item.
+      // At index 0 of the array is the data label (sentRates[0] == "TCP Sent" and receivedRates[0] == "TCP Received").
+      // This is why we skip the first element in each array.
+      dataSent = this.props.sentRates[1].slice(1);
+      dataReceived = this.props.receivedRates[1].slice(1);
+    }
+
+    let minSent: number = 0,
+      maxSent: number = 0,
+      minReceived: number = 0,
+      maxReceived: number = 0;
+
+    if (dataSent.length > 0) {
+      minSent = Math.min(...dataSent);
+      maxSent = Math.max(...dataSent);
+    }
+    if (dataReceived.length > 0) {
+      minReceived = Math.min(...dataReceived);
+      maxReceived = Math.max(...dataReceived);
+    }
 
-  private renderNoTrafficLegend = () => {
     return (
       <div>
-        <Icon type="pf" name="info" /> No traffic logged.
+        <Icon name="square" style={{ color: PfColors.Blue }} /> Sent: {minSent.toFixed(2)} / {maxSent.toFixed(2)}{' '}
+        bytes/second
+        <br />
+        <Icon name="square" style={{ color: PfColors.Green }} /> Received: {minReceived.toFixed(2)} /{' '}
+        {maxReceived.toFixed(2)} bytes/second
```

Comment:
Is this really bytes / second?
I think we could us `B/s` and even do some math to convert to `kB/s` when needed or we are going to end up with a big number here.

Reply:
Yes, it's bytes/sec.
And yes, probably it's good to move to the next unit if numbers are too big. Although, I don't know if we should do the same for the data for the sparkline... WDYT?

Reply:
For the sparkline it might be a little complicated. I would say that is good if you can have the `b/s` value in the graph, but when the user hover it shows the "most appropriate" unit it.

Reply:
@josejulio  I updated the code. Now the units are formatted, both for the min/max stats and also in the sparkline tooltip.